src/app/collective-rewards/rewards/backers/RewardsCard.tsx

```diff
@@ -58,25 +60,27 @@ const TokenRewardsMetrics: FC<TokenRewardsMetricsProps> = ({
   })
 }
 
-type RewardsCardProps = Omit<RewardDetails, 'gauge'> & {
-  title: string
-  'data-testid': string
-  rewards: Rewards[]
-  tooltip?: TooltipProps
+type RewardsCardProps = MetricsCardProps & {
+  rewardDetails: Omit<RewardDetails, 'gauge'> & {
+    rewards: Rewards[]
+  }
+  titleDetails: MetricsCardTitleProps
 }
 
 export const RewardsCard: FC<RewardsCardProps> = ({
-  title,
-  'data-testid': dataTestId,
-  tokens: { rif, rbtc },
-  tooltip,
+  rewardDetails: {
+    tokens: { rif, rbtc },
+    ...restOfRewardDetails
+  },
+  titleDetails,
+  dataTestId,
   ...rest
 }) => {
   return (
-    <MetricsCard borderless>
-      <MetricsCardTitle title={title} data-testid={dataTestId} tooltip={tooltip} />
-      <TokenRewardsMetrics {...rest} token={rif} />
-      <TokenRewardsMetrics {...rest} token={rbtc} />
+    <MetricsCard borderless dataTestId={dataTestId} {...rest}>
+      <MetricsCardTitle data-testid={dataTestId} {...titleDetails} />
```

Comment:
What's the point of using th same `data-testid` attribute in different components?

Reply:
each component will prepend it to its own data-test-id so that the tests can track the composition

Reply:
It makes sense since it's used as prefix.